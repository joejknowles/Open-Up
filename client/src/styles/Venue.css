.Venue {
  width: 40%;
  margin: auto;
  position: relative;
}

.venue-heading {
  text-decoration: underline;
}

.bookLink {
  position: absolute;
  right: 0;
  border-style: solid;
  border-width: thin;
  color: black;
  text-decoration: none;
  padding: 0.2em;
  -webkit-box-shadow: 6px 6px 20px -8px rgba(0,0,0,0.75);
  -moz-box-shadow: 6px 6px 20px -8px rgba(0,0,0,0.75);
  box-shadow: 6px 6px 20px -8px rgba(0,0,0,0.75);
}
