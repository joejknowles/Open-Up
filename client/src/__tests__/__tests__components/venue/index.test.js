import Venue from '../../../components/venue';

import React from 'react';
import ReactTestUtils from 'react-addons-test-utils';

const shallowRenderer = ReactTestUtils.createRenderer();

it('renders Venue as before', () => {
  const venue = shallowRenderer.render(<Venue />);
  expect(venue).toMatchSnapshot();
});
