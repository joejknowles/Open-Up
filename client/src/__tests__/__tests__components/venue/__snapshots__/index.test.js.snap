exports[`test renders Venue as before 1`] = `
<div
  className="Venue">
  <Link
    className="bookLink"
    onlyActiveOnIndex={false}
    style={Object {}}
    to="/venue/book">
    BOOK HERE
  </Link>
  <h3
    className="venue-heading">
    Venue 1
  </h3>
  <Unknown
    className="venue-image" />
  <p
    className="venue-info">
    <em>
      Info:
    </em>
     This is general information about the venue.
  </p>
</div>
`;
